src/NuGet.Core/NuGet.Packaging.Core/IPackageCoreReader.cs

```diff
@@ -28,10 +28,9 @@ public interface IPackageCoreReader : IDisposable
         NuGetVersion GetMinClientVersion();
 
         /// <summary>
-        /// Gets the <see cref="PackageType"/> of the package.
+        /// Gets zero or more package types from the .nuspec.
         /// </summary>
-        /// <returns>The <see cref="PackageType"/>.</returns>
-        PackageType GetPackageType();
+        IEnumerable<PackageType> GetPackageTypes();
```

Comment:
Perhaps expose this as `IReadOnlyList<PackageType>` if you have to. You can avoid the enumeration costs of `Count()` \ `First()` etc.